app/src/App/App.tsx

```diff
@@ -625,7 +625,7 @@ const App: React.FC = () => {
             />
             <PublicComponent exact path={APP_CALLBACK_PATH} component={CallbackComponent} />
             {/* tslint:enable jsx-no-lambda */}
-            <PrivateComponent
+            <PublicComponent
```

Comment:
why this change?

Reply:
PrivateComponent has enforced the use of roles to secure components and we don't need roles to log out a user.